bodhi/server/scripts/clean_old_mashes.py

```diff
@@ -33,6 +33,12 @@
 @click.command()
 @click.version_option(message='%(version)s')
 def clean_up():
+    """Delete any repo mashes that are older than the newest 10 from each repo series."""
+    _clean_up_()
+
+
+# Helper function used in auto clean composes (masher.py)
+def _clean_up_():
```

Comment:
Let's give this a public name (no leading underscore). To avoid clashing with the other function, let's call this ```remove_old_composes()```.